packages/viewer/src/IViewer.ts

```diff
@@ -91,7 +91,7 @@ export interface ViewerEventPayload {
 export type SpeckleView = {
   name: string
   id: string
-  view: { origin: Vector3Like; target: Vector3Like }
+  view: { name: string; origin: Vector3Like; target: Vector3Like }
```

Comment:
@andrewwallacespeckle Confirm w/ @AlexandruPopovici. Even if there is such a prop, it might be optional or nullable

Reply:
Seems like it does have a `name` https://github.com/specklesystems/speckle-sharp/blob/b2f9f49cde910b080a5aa118a66c3a40452f8693/Objects/Objects/BuiltElements/View.cs#L11

Reply:
Thanks for checking @AlexandruPopovici. Can you let me know when you have had a look? It would be good to get this out soon as views are currently broken without this.